### DISCUSSION TITLE
The data is not up to date

### DISCUSSION THREAD

**Discussion by dcaribou** [dcaribou/transfermarkt-datasets#299] [FAQ] [answered]
I was expecting certain dataset to contain up to date data, but it seems to be missing.

**Comment by dcaribou** [comment, marked as answer]
While the dataset should update on schedule (see https://github.com/dcaribou/transfermarkt-datasets/discussions/214), new data may occasionally contain errors that need to be debugged manually, and hence the pipeline is stopped for safety.  The status os the pipeline is displayed by the badge below.

→ ![Build Status](https://github.com/dcaribou/transfermarkt-datasets/actions/workflows/build.yml/badge.svg)

If the status is 🔴 , it means I am aware of the issue and I will review when possible. If the status is 🟢 , and you still find data to be missing, please **add a comment in this discussion**.

**Comment by mortgad** [comment]
The appearances.csv file is still missing data

**Reply by dcaribou** [reply]
Could you try now?

**Reply by mortgad** [reply]
Seems to be working now.

What causes the lack of updates? Would running the scraper myself get the updated data?

**Reply by dcaribou** [reply]
It's usually a change on transfermarkt side that breaks the scraper. Running it yourself would not help in most of the cases, as the scraper needs to be fixed first.

**Comment by mortgad** [comment]
Hi David

It seems the pipeline is broken again, as the latest date in the appearances-file is the 28th of Nov.
Do you think it is possible to have updated data before Friday?

**Reply by dcaribou** [reply]
Hi @mortgad,
Thanks for raising. The updates should be coming through in the next minutes (https://github.com/dcaribou/transfermarkt-datasets/commit/474b6e99dfac05c07f5feafa7a4fb0a2570f1810).

**Reply by mortgad** [reply]
Thank you for the fix last time!

It seems that appearances from four matches from the latest Spanish game week are missing from the appearances.csv file

Do you have time to take a look at the pipeline?

**Comment by mortgad** [comment]
Hi David,

Fixtures from last week are missing from the appearances file.
Can you check the pipeline?
